server/src/definition/ModelDefinitionProvider.ts

```diff
@@ -85,9 +87,8 @@ export class ModelDefinitionProvider implements DbtNodeDefinitionProvider {
       );
   }
 
-  searchModelDefinition(dbPackage: string, model: string, dbtModels: ManifestModel[], modelSelectionRange: Range): DefinitionLink[] | undefined {
-    const modelId = `model.${dbPackage.slice(1, -1)}.${model.slice(1, -1)}`;
-    const foundModel = dbtModels.find(m => m.uniqueId === modelId);
+  searchModelDefinition(model: string, dbtModels: ManifestModel[], modelSelectionRange: Range, dbPackage?: string): DefinitionLink[] | undefined {
+    const foundModel = dbtModels.find(m => m.name === model && (dbPackage === undefined || m.packageName === dbPackage));
```

Comment:
Do we have all installed package's models inside manifest.json with specified `package_name`?
So `dbPackage === undefined` means that we find only by name. What will happen if our model will have the same name as model from installed package? Is it possible?

Reply:
Good note! According to dbt compile result project model has more priority than package model.